### PR TITLE
docs: fix invalid relative link in getting started page details section

### PR DESCRIPTION
**Pull request by salitha-pathi**
## PR Checklist

- [ ] Addresses an existing open issue
- [ ] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [ ] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

Hey there! 👋

In the previous update to the link in the "Getting Started" page, it was pointing from the current directory, but it actually needed to go one directory up. My apologies for the mistake.

This PR fixes that issue by adjusting the link appropriately.

Thanks for reviewing! 🚀

### CODE REVIEW

**Comment by salitha-pathi** [comment]
> this was already fixed.

still cannot see on the server. going to invalid page

**Comment by bradzacher** [comment]
@salitha-pathi you'll likely need to force refresh the page.
The page caches things with a service worker for offline use so it can take a few refreshes to update sometimes.

**Comment by bradzacher** [comment]
Oh wait

File: docs/Getting_Started.mdx [hunk 1]
```diff
@@ -72,7 +72,7 @@ The [`config`](./packages/TypeScript_ESLint.mdx) helper sets three parts of ESLi
 
 - [Parser](https://eslint.org/docs/latest/use/configure/parser): set to [`@typescript-eslint/parser`](./packages/Parser.mdx) so ESLint knows how to parse the new pieces of TypeScript syntax.
 - [Plugins](https://eslint.org/docs/latest/use/configure/plugins): set to [`@typescript-eslint/eslint-plugin`](./packages/ESLint_Plugin.mdx) to load [rules listed in our _Rules_ page](./Rules).
```
Comment (auvred) [inline docs/Getting_Started.mdx:74]:
```suggestion
- [Plugins](https://eslint.org/docs/latest/use/configure/plugins): set to [`@typescript-eslint/eslint-plugin`](./packages/ESLint_Plugin.mdx) to load [rules listed in our _Rules_ page](/rules).
```

Let's fix one more broken link in this PR? 

https://github.com/typescript-eslint/typescript-eslint/issues/8480

**Review by auvred — CHANGES_REQUESTED** [review]

**Review by bradzacher — APPROVED** [review]

**Pull request merged**